ui/lib/pki/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
+/* eslint-disable ember/avoid-leaking-state-in-ember-objects */
+/* eslint-disable node/no-extraneous-require */
+'use strict';
+
+const EngineAddon = require('ember-engines/lib/engine-addon');
```

Comment:
Also taken from the quickstart guide, but could we do something like this instead?

```
const { buildEngine } = require('ember-engines/lib/engine-addon');

module.exports = buildEngine({
  name: 'pki',
  lazyLoading: {
    enabled: false
  }
});